.rubocop.yml

```diff
@@ -47,3 +50,6 @@ Metrics/BlockLength:
 
 Rails/SkipsModelValidations:
   Enabled: false
+
+Rails/LexicallyScopedActionFilter:
+  Enabled: false
```

Comment:
Personally I prefer not to disable cops at this level and disable within the file if they are not applicable.